npm-react-tools/README.md

```diff
@@ -45,6 +46,7 @@ option | values | default
 `stripTypes` | `true`: strips out type annotations | `false`
 `es6module` | `true`: parses the file as an ES6 module | `false`
 `nonStrictEs6module` | `true`: parses the file as an ES6 module, except disables implicit strict-mode (i.e. CommonJS modules et al are allowed) | `false`
+`target` | `"es3"`: ECMAScript 3<br>`"es5"`: ECMAScript 5| `"es5"`
```

Comment:
Right, but is it good style?  Is it what we want to do here.  My understanding (which is admittedly limited) is that we want to avoid HTML in the markdown files whenever possible.  In the case of a linebreak, it seems avoidable, but maybe it's necessary/desirable here because we're in a table.